src/core/pv.jl

```diff
@@ -82,7 +82,7 @@
     If `azimuth` is not provided, then it is set to 180 if the site is in the northern hemisphere and 0 if in the southern hemisphere.
 
 """
-struct PV <: AbstractTech
+mutable struct PV <: AbstractTech
```

Comment:
fine for now but we should be trying to avoid making structs mutable

Reply:
@hdunham do you want to make an issue for this?